core/Container/StaticContainer.php

```diff
@@ -13,7 +13,8 @@
 /**
  * This class provides a static access to the container.
  *
- * @deprecated This class is introduced only to keep BC with the current static architecture. It will be removed in 3.0.
+ * @internal
```

Comment:
I was just wondering how to delete all instances of code that depended on it, do you have any idea or an example of good migrations StaticContainer to another things ?

Reply:
I guess the solution would be using dependency injection everywhere. We already have that in some places, but it's not yet usable everywhere...